.php_cs.dist

```diff
@@ -0,0 +1,280 @@
+<?php
+$finder = array_reduce(
+    [
+        __DIR__ . '/src/',
+        __DIR__ . '/bin/',
+        __DIR__ . '/tests/'
+    ],
+    function (PhpCsFixer\Finder $finder, $dir) {
+        return $finder->in($dir);
+    },
+    PhpCsFixer\Finder::create()->ignoreUnreadableDirs()
+)->notPath('/stubs/i')->notName('CallMap.php');
+$rules = [
+    '@PSR2' => true,
+    'array_syntax' => [
+        'syntax' => 'short',
+    ],
+    'binary_operator_spaces' => [
+        'align_double_arrow' => false,
+        'align_equals' => false,
+    ],
+    'blank_line_after_opening_tag' => false,
+    'blank_line_before_return' => true,
+    'cast_spaces' => false,
+    'class_keyword_remove' => false,
+    'combine_consecutive_unsets' => true,
+    'concat_space' => [
+        'spacing' => 'one',
+    ],
+    /* // to be removed/altered if vimeo/psalm ever goes php:^7.0
+    'declare_equal_normalize' => [
+        'space' => 'none',
```

Comment:
php 5.3 supports `declare(encoding='...');`, so this shouldn't be an issue, yes?